### PR TITLE
Improved `/help` output

### LINKED ISSUE #30: Add the GitHub repo link to the /help command
... and replace the user activity with something else.

### PR DESCRIPTION
**Pull request by AverageHelper**
Listing commands is hard, especially when permissions get involved.

In this PR, I gave up trying to replicate Discord's command permissions resolution algorithm, and instead direct users to Discord's own UI for that. That UI sucks, especially in mobile contexts, but that's Discord's problem to fix, and users should go to them to ask for it fixed.

The new output also includes a link to this repo, and the current release version. (Closes #30)

Screen Shot 2022-09-30 at 12 59 23

I also removed the deprecated `replyPrivately` and `isGuildedCommandContext` functions. Their behavior was replaced in #25 with `replyWithPrivateMessage` and `context.source === 'guild'`, respectively. Be sure to update your branch(s) if you use either of these functions.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -63,17 +63,6 @@ export async function _main(): Promise<void> {
...
-		client.user.setActivity({
-			// ActivityType.Playing looks like crap, but it's the only way
-			// to show a custom multiline string on the bot's user profile.
-			// I'd put the URL in "About Me", but Discord seems to delete
-			// those sometimes, and I'd like to stay on Discord's good side.
-			type: ActivityType.Playing,
-			name: 'Source: github.com/BYU-CS-Discord/CSBot',
-			url: 'https://github.com/BYU-CS-Discord/CSBot',
-		});
-
```
Comment (JstnMcBrd) [inline src/main.ts:76]: What should we replace this with? Preferably the bot should have _something_ for its activity.

With my bots, I like to have their activities direct users to use the /help command. Thoughts?

Comment (AverageHelper) [reply src/main.ts:76]: No idea

Comment (ZYancey) [reply src/main.ts:76]: "Playing /help for info" is fine. I think people get it and its what almost every bot uses unless its something funny.

Comment (AverageHelper) [reply src/main.ts]: Added

**Review by JstnMcBrd — APPROVED** [review]

**Review by ZYancey — APPROVED** [review]

**Pull request merged**